### ISSUE TITLE
Unable to read files in `Iowa_profiles_2010`

### ISSUE THREAD

**Issue by padilla410** [USGS-R/lake-temperature-model-prep#275]
There is something weird going on with the [2010 data](https://drive.google.com/drive/folders/1dQPlto0d3LAxx3m0soHMQf-_YxlBC0AF) from Iowa DNR. When I try to use `readxl::read_xls` to read the data in I get the following error:
```r
Error: 
  filepath: C:\Users\jpadilla\AppData\Local\Temp\2\RtmpgzNlt7\Iowa_DNR_LimnoProfiles_2000-2020\Iowa_profiles_2010\L_259_D_08-02-2010_SI_2010214011.xls
  libxls error: Unable to open file
```

If you grab one of the files from the link above, this code snippet will reproduce the error I am seeing:
```r
dat_2010 <- readxl::read_xls("your_full_file_path_here")
```
I also get prompt when I try to open in Excel 365 (the file will open though):
![image](https://user-images.githubusercontent.com/15007294/148600734-08765725-e735-473f-bc07-13e298b36b48.png)

My first thought it that these are actually `csv` files that are mislabeled as `xls` but I do not know of a great way to programmatically fix this. I have tried the following:
* manually changing the file extension from `xls` to `csv` - Same excel error as above. I can read into R using `read_csv` but the `xls` tags come along for the ride and data is unusable
* manually changing the file extension from `xls` to `xlsx` - Same excel error as above. Errors when trying to read into R using `readxl::read_xlsx`
* opening it up in Excel and then trying to save it. When I do that I get this prompt:
![image](https://user-images.githubusercontent.com/15007294/148602255-658547cb-6c3f-4780-971c-19b33f5f49ef.png)

I do not have problems reading files from any of the other files within [`Iowa_DNR_LimnoProfiles_2000-2020`](https://drive.google.com/drive/folders/1NNkb-QrGrsxcjKsN4eiB56du-eay-phx)

**Comment by jread-usgs** [comment]
Interesting. I get the same (or very similar) errors here on mac. But, like you, am able to open the file in excel if I wave away the warning. 

when trying to save after I open, I get a default file type of `.htm`
![image](https://user-images.githubusercontent.com/2349007/148607831-5b5bc6b1-6d2d-4bb6-a480-efe35c228fd5.png)


Thoughts on saving as an .xls and replacing the file on drive?

**Comment by lindsayplatt** [comment]
There are hundreds of these files, so the manual step is not ideal

**Comment by jread-usgs** [comment]
oh :( I misread that and thought it was only one corrupt file.

**Comment by lindsayplatt** [comment]
This happened to me this summer with some lab data. I had the owner re-save and re-upload. That doesn't really feel like an option here?

**Comment by jread-usgs** [comment]
Yay `rvest` !
![image](https://user-images.githubusercontent.com/2349007/148609799-ea814201-c58a-4dc0-9efb-ea170631e8a7.png)

These seem to be html :( but at least they can be read

**Comment by lindsayplatt** [comment]
huh that is incredibly strange - nice solution! I wonder how that happens to an excel file

**Comment by padilla410** [comment]
This is great news, because I am also having this issue on a subset of the data because I am also running into this on a subset of the 2011-2016 data. Thanks everybody!

I'll add in a little function to ID all these weirdos and then process them together.

**Comment by padilla410** [comment]
Using the advice above I was able to resolve this issue. Iowa DNR data was merged in PR #284
